patches/components-crash-core-app-breakpad_linux.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/components/crash/core/app/breakpad_linux.cc b/components/crash/core/app/breakpad_linux.cc
+index 9e850c0380b8a856dace5f68c80dd11a5b95a671..2726278ed03fa6c474ef96853f02bb0c6317c5c8 100644
+--- a/components/crash/core/app/breakpad_linux.cc
++++ b/components/crash/core/app/breakpad_linux.cc
+@@ -1458,7 +1458,7 @@ bool IsValidCrashReportId(const char* buf, size_t bytes_read,
+   return my_strcmp(buf, "_sys_cr_finished") == 0;
+ #else
+   for (size_t i = 0; i < bytes_read; ++i) {
+-    if (!my_isxdigit(buf[i]))
++    if (!my_isxdigit(buf[i]) && buf[i] != '-')
```

Comment:
it would be better to do `if (buf[i] != '-')` on the next line so you have an added line instead of a changed line and also this should be a define so it's changeable without updating the patch